src/infrastructure/messaging/message.types.ts

```diff
@@ -0,0 +1,35 @@
+export enum MessageType {
+    Unknown,
+    RentalRequested,
+    RentalAccepted,
+    RentalDeclined,
+    CustomerCreated,
+    CustomerUpdated,
+    CustomerDeleted
+  }
+
+// tslint:disable-next-line:no-namespace
+export namespace MessageType {
+    export function toString(type: MessageType) {
+      return messageTypeName.get(type);
+    }
+
+    export function parse(type: string): MessageType {
+      for (const iterator of messageTypeName.entries()) {
+        if (iterator['1'] === type) {
+          return iterator['0'];
+        }
+      }
+
+      return MessageType.Unknown;
+    }
+  }
+
+const messageTypeName = new Map<MessageType, string>([
```

Comment:
Remove all "Event" of MessageTypes. See all other services

Reply:
fixed